docs/config/header.html

```diff
@@ -90,7 +99,7 @@
           <li style="min-width: 130px;">
             <a href="modules.html">
               <i class="octicon octicon-code"></i>
-              <span>Reference</span>
+              <span>Modules</span>
```

Comment:
no

Reply:
Both the tab in the navigation bar and the first item in its dropdown menu read _Reference_, so the hierarchy doesn't make sense. _Reference_ is also very general, the whole documentation is a reference, so attempted to find a better name / used the default. Thoughts?

Reply:
Btw. are we sticking with this documentation format / style (for the foreseeable future)? If you're planning to switch anyway it's useless to discuss these details.

Reply:
I see what you're saying. Maybe "Overview"? I don't like modules because C++ modules, python modules, charm CI modules, etc. Modules is just too generic.

Reply:
I don't know what we would switch to, to be honest. We are basically stuck with Doxygen at the moment

Reply:
Yeah these are not modules. How about _Topics_?

Reply:
I'm fine with that, can't think of anything better :)